### ISSUE TITLE
replace specific utf-8 characters with ascii

### ISSUE THREAD

**Issue by NCIEVS** [NCIEVS/nci-protege5#458]
This is an enhancement request to convert a number of valid utf8 characters into ascii (utf8 enforcement is already done in the GUI as well as in batch files).  Essentially, there are a few valid utf8 characters that are best replaced with facsimile ascii characters.  For instance replace the non-breaking space with a regular space.  

We are already doing these character replacements in batch loads/edits (**#144**) and we want to extend the same approach to cover the various text entry dialogs in the GUI.  We'd use the same [**map**](https://github.com/NCIEVS/nci-edit-tab/blob/master/src/main/resources/symbolmap.data) that we are using for batch edits.  

The check & replacement might best be made at Save time.

To check with editors:  if there is a replacement that needs to be done, would the replacement be done and the editor notified with an informational pop-up dialog, or should the replacement be made silently?

**Comment by bdionne** [comment]
@NCIEVS - it seems like we can use the same replacement table for this?

**Comment by NCIEVS** [comment]
@bdionne , yes, then if a new character needs to be dealt with we'd just add it to the map.
